pages/sessions.jsx

```diff
@@ -232,7 +227,7 @@ var SpacesInfo = [
 
 var SessionsPage = React.createClass({
   spaces: SpacesInfo,
-  componentWillMount: function() {
+  render: function() {
```

Comment:
@alanmoo 😭  l know having `.map` in `render` isn't a great solution but fixing that `react-router` context error is taking me forever... and I don't how much longer it will take me still. 

I've tried moving `.map` to `componentDidMount`... same `Warning: owner-based and parent-based contexts differ (values: function (props, context)` and `TypeError: this.context.router is undefined` errors occurred.

Creating a `ContextWrapper` as suggested in https://github.com/nhunzaker/react-router/blob/c6fa1a3d8baafce62eaf3c41af966788c7a250c1/modules/components/ContextWrapper.js didn't help. Same errors.

Since there aren't a ton of items in the array (won't affect performance much), can we merge this PR first? I'll create a ticket to follow up this react-router context problem.  I thinks we should not spend any more time on this (😷) since this problem is likely to go away when we update our dependencies (or rewrite the site from scratch next year).